tests/js/server/aql/aql-modify-noncluster.js

```diff
@@ -2386,6 +2428,20 @@ function ahuacatlUpdateSuite () {
       }
     },
 
+///////////////////////////////////////////////////////////////////////////////
+/// @brief test update with empty object
+////////////////////////////////////////////////////////////////////////////////
+
+    testUpdateEmpty3: function () {
+        const actual = AQL_EXECUTE(`FOR doc IN ${cn3} UPDATE doc WITH {} IN ${cn3} RETURN {old: OLD, new: NEW}`);
+        const res = actual.json;
+        for (let i = 0; i < res.length; ++i) {
```

Comment:
Please add an assertion for `res.length` here, so we know that the test actually works on some documents.

Reply:
Addressed